doc/build/changelog/unreleased_20/8765.rst

```diff
@@ -0,0 +1,10 @@
+.. change::
+    :tags: usecase, postgresql
+    :tickets: 8765
+
+    Complementing :ticket:`8690`, new comparison methods such as
```

Comment:
**mike bayer** (zzzeek) wrote:

perfect!

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4182